packages/mobile/src/import/saga.ts

```diff
@@ -74,7 +75,12 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
     // Set redeem invite complete so user isn't brought back into nux flow
     yield put(redeemInviteSuccess())
     yield put(refreshAllBalances())
-    navigate(Screens.VerificationEducationScreen)
+
+    // Check if the account was verified
+    const isVerified = yield call(checkVerification)
+
+    navigate(isVerified ? Screens.ExchangeHomeScreen : Screens.VerificationEducationScreen)
```

Comment:
You should navigate to WalletHome, not ExchangeHome

Reply:
Done!